wc-api-dev-class.php

```diff
@@ -101,6 +101,10 @@ public function includes() {
 		include_once( dirname( __FILE__ ) . '/api/class-wc-rest-dev-shipping-methods-controller.php' );
 		include_once( dirname( __FILE__ ) . '/api/class-wc-rest-dev-payment-gateways-controller.php' );
 
+		if ( class_exists( 'MailChimp_Woocommerce' ) ) {
```

Comment:
We should add a comment above this line that this endpoint is for Calypso/Store on .com work only and should never be merged to core.